server/lib/snippets.js

```diff
@@ -0,0 +1,312 @@
+                  - `l10n` - [optional] Set it to true if the `value`
+                             property of this object is a localization key.
+                             Defaults to false.
+*/
+
+const HTML = [{
+  id: "snippet-htmlComment",
+  name: "snippetHTMLComment",
+  title: "snippetHTMLCommentTitle",
+  data: {
+    value: "snippetHTMLCommentData",
+    l10n: true
+  }
+}, {
+  id: "snippet-table",
+  name: "snippetHTMLTable",
+  title: "snippetHTMLTableTitle",
+  data: {
+    value: '<table>\n' +
+           '  <tr>\n' +
+           '    <th></th>\n' +
+           '    <th></th>\n' +
+           '  </tr>\n' +
+           '  <tr>\n' +
+           '    <td></td>\n' +
+           '    <td></td>\n' +
+           '  </tr>\n' +
+           '</table>\n'
+  }
+}, {
+  id: "snippet-definitionList",
+  name: "snippetHTMLDefinitionList",
+  title: "snippetHTMLDefinitionListTitle",
+  data: {
+    value: '<dl>\n' +
+           '  <dt></dt>\n' +
+           '  <dd></dd>\n' +
+           '  <dt></dt>\n' +
+           '  <dd></dd>\n' +
+           '</dl>\n'
+  }
+}, {
+  id: "snippet-orderedList",
+  name: "snippetHTMLOrderedList",
+  title: "snippetHTMLOrderedListTitle",
+  data: {
+    value: '<ol>\n' +
+           '  <li></li>\n' +
+           '  <li></li>\n' +
+           '  <li></li>\n' +
+           '</ol>\n'
+  }
+}, {
+  id: "snippet-unorderedList",
+  name: "snippetHTMLUnorderedList",
+  title: "snippetHTMLUnorderedListTitle",
+  data: {
+    value: '<ul>\n' +
+           '  <li></li>\n' +
+           '  <li></li>\n' +
+           '  <li></li>\n' +
+           '</ul>\n'
+  }
+}, {
+  id: "snippet-form",
+  name: "snippetHTMLForm",
+  title: "snippetHTMLFormTitle",
+  data: {
+    value: "snippetHTMLFormData",
+    l10n: true
+  }
+}, {
+  id: "snippet-script",
+  name: "snippetHTMLScript",
+  title: "snippetHTMLScriptTitle",
+  data: {
+    value: "snippetHTMLScriptData",
+    l10n: true
+  }
+}, {
+  id: "snippet-internal-stylesheet",
+  name: "snippetHTMLInternalStylesheet",
+  title: "snippetHTMLInternalStylesheetTitle",
+  data: {
+    value: '<style>\n' +
+           '  h1 {color:red;}\n' +
+           '  p {color:blue;}\n' +
+           '</style>\n'
+  }
+}, {
+  id: "snippet-external-stylesheet",
+  name: "snippetHTMLExternalStylesheet",
+  title: "snippetHTMLExternalStylesheetTitle",
+  data: {
+    value: "snippetHTMLExternalStylesheetData",
+    l10n: true
+  }
+}, {
+  id: "snippet-video",
+  name: "snippetHTMLVideo",
+  title: "snippetHTMLVideoTitle",
+  data: {
+    value: "snippetHTMLVideoData",
+    l10n: true
+  }
+}, {
+  id: "snippet-audio",
+  name: "snippetHTMLAudio",
+  title: "snippetHTMLAudioTitle",
+  data: {
+    value: "snippetHTMLAudioData",
+    l10n: true
+  }
+}, {
+  id: "snippet-mmt",
+  name: "snippetHTMLMetaMobile",
+  title: "snippetHTMLMetaMobileTitle",
+  data: {
+    value: '<meta name="viewport" content="width=device-width, initial-scale=1.0">'
+  }
+}];
+
+const CSS = [{
```

Comment:
darn `CSS` is an object on `window`. @twigz20 I would advise that you make all of these lowercase throughout your code.